explainer/Explainer11.md

```diff
@@ -0,0 +1,287 @@
+    The subprotocol clarifies which of several possible mechanisms in HTTP
+    are used to provide "push" notifications.
+  - Each message pushed by the Thing is a string value.
+
+
+## What you can do with a Thing Description (TD)?
+
+Based on such information in a TD a WoT stack such as designed based on the [W3C Web of Things Scripting API](https://w3c.github.io/wot-scripting-api/) can simple interpret the TD content and abstract all the communication details. Developer just working with the **properties, actions**, and **events** paradigm and do not need to take care protocol specifics such as the HTTP method, the resource address, port number, etc.. The following video gives an inside about this aspect:
+
+[![IMAGE ALT TEXT](http://img.youtube.com/vi/lt_P2BU8e3I/0.jpg)](http://www.youtube.com/watch?v=lt_P2BU8e3I "TD usage with a programming API")
+
+This example also shows that TDs can be used to easily onboard Things into any IoT ecosystems (e.g., IoT cloud services, edge systems, etc.). TDs helps to understand what the Things offers and how the offered data and functions can be bound to applications processes (e.g., creating service mash-ups, dashboards, etc.). The following example shows how TD are used in a node-red application:
+
+[![IMAGE ALT TEXT](http://img.youtube.com/vi/oAcYbJ6P9bU/0.jpg)](http://www.youtube.com/watch?v=oAcYbJ6P9bU "TD usage in node-red")
+
+TD also helps to mange your IoT system that typically consist of a heterogeneous device landscape based on different vendors and technologies that are used (e.g., HTTP, MQTT, Modbus, etc). Directories can be used to manage the TDs (e.g., create, search, etc). [The W3C WoT Discovery](https://w3c.github.io/wot-discovery/) provides details about a standardized how the interface of such directories can look like.  
+
+
+## What else is possible with the Thing Description specification?
+Besides the definitions of how a Thing description should be designed, there are other features provided around the TD, such as the Protocol Binding concept, TD context extensions, and Thing Model definitions. 
+
+### Protocol Binding
+Thing Description is not limited to http based interfaces as shown in the example above. In general, WoT is a protocol agnostic approach and provides a common mechanism to define how specific protocols such as MQTT, HTTP, CoAP or Modbus can be mapped to the WoT’s interaction properties-action-event abstraction within the Thing Description forms definition. More details is provide in the [Protocol Binding section](https://w3c.github.io/wot-thing-description/#protocol-bindings).  
+ 
+
+### TD Context Extensions
+Through JSON-LD serialization, the WoT Thing Description provides the ability to add context knowledge from additional namespaces. This mechanism can be used to enrich the Thing Description instances with additional (e.g., domain-specific) semantics. It can also be used to import additional Protocol Bindings or new security schemes in the future. 
```

Comment:
context knowledge from -> vocabulary and concepts defined in

"in the future": not really needed, sounds like we need a new feature.  Maybe just delete ("... can be used to import additional Protocol Bindings or new security schemes.")

Reply:
done